jw-react-tool/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SearchBar from '../SearchBar/SearchBar';
+import PostContainer from '../PostContainer/PostContainer';
+import dummyData from '../dummy-data';
+
+
+class PostsPage extends React.Component {
+    constructor () {
+        super();
+        this.state = {
+            dummy: dummyData,
+            filtered: [1, 2, 3, 4, 5],
+            search:''
+        }
+    }
+    
+    componentDidMount() {
+        this.setState({ dummy: dummyData})
+    }
+    
+    
+    
+    handleSearch = e => {
+        this.setState({ search: e.target.value});
+    
+        let filteredDummy = this.state.dummy.filter(
+            (contact) => {
+                return contact.username.indexOf(this.state.search) !== -1;
```

Comment:
```suggestion
                return contact.username.toLowerCase().indexOf(this.state.search.toLowerCase()) !== -1;
```
to make it case-insensitive :)